src/app/containers/Ad/Amp/index.jsx

```diff
@@ -78,9 +78,19 @@ export const AMP_ACCESS_FETCH = service => {
 const AmpAd = ({ slotType }) => {
   const { ads, dir, script, service } = useContext(ServiceContext);
   const label = pathOr('Advertisement', ['advertisementLabel'], ads);
+  const mpuLabel =
```

Comment:
To ensure we do not repeat ourselves, we could create a utility function that does this logic for us, and then we can also add unit tests to this function.

You could create a utils function called getAdsAriaLabel and combine the logic of mpuLabel and ariaLabel.